### ISSUE TITLE
XRDP problem in ubuntu of win11 linux subsystem

### ISSUE THREAD

**Issue by ikarlmarx** [neutrinolabs/xrdp#2960]
### xrdp version

0.9.20

### Detailed xrdp version, build options

~~~
I just installed xrdp in ubuntu with mate-desktop, which is a sub system in win11.   
But it will quit quickly when I log in.  There are two files, wayland-0 and wayland-0.lock , in //run/user/1000 preventing the logging.
When I delete these two files, I can log in properly.

Another problem is,  two xrdps in different ubuntus as sub systems of win11 cannot work together, with one's port is 3390 and the other's port is 3391.  If I want to log in one of them , I have to shut down anonther one.

Any help?

Thank you.
~~~

### Operating system & version

Ubuntu 22.04

### Installation method

dnf / apt / zypper / pkg / etc

### Which backend do you use?

xorgxrdp

### What desktop environment do you use?

ubuntu-mate

### Environment xrdp running on

A sub system in win11

### What's your client?

remote-desktop in win11

### Area(s) with issue?

Network

### Steps to reproduce

sudo install ubuntu-mate-desktop
sudo install xrdp

config xrdp

connect from win 11 through remote-desktop

### ✔️ Expected Behavior

connected

### ❌ Actual Behavior

quit quickly

### Anything else?

No

**Comment by matt335672** [comment]
@ikarlmarx - this is almost certainly in [the FAQ](https://github.com/neutrinolabs/xrdp/wiki/Tips-and-FAQ#why-cant-i-log-the-same-user-on-on-the-graphical-console-and-over-xrdp-at-the-same-time)

If you've got files `wayland-0` and `wayland-0.lock`  in `/run/user/1000` you will be logged in on the Ubuntu console.

**Comment by ikarlmarx** [comment]
> @ikarlmarx - this is almost certainly in [the FAQ](https://github.com/neutrinolabs/xrdp/wiki/Tips-and-FAQ#why-cant-i-log-the-same-user-on-on-the-graphical-console-and-over-xrdp-at-the-same-time)
> 
> If you've got files `wayland-0` and `wayland-0.lock` in `/run/user/1000` you will be logged in on the Ubuntu console.

Yes. I added a new user.  Now it works well.
Thank you.

**Comment by ikarlmarx** [comment]
Is it possible that two XRDPs work together in a same Win 11 as sub system?

**Comment by matt335672** [comment]
I can't see why not.

What happens when you try it?

**Comment by CamiloCorredor** [comment]
I have a similar issue, my SO is Ubuntu 22.04, i have installed xrdp as "apt install xrdp" and start the server. I have reviewed the state of the connection doing a ping with my IP and port from Windows 11 and the connection is enable. However, when I try to connect with RDP on Windows, I got an error with the library libxrdp-vnc.so, it seems, it is not installed or the route of this SO file is lost and I should indicate the real file's route on xrdp.ini. Until this momento, i had couldnt install o find the library to access to remote desktop. Any idea to fix and achieve the connection?
